components/eamxx/CMakeLists.txt

```diff
@@ -204,11 +200,6 @@ endif()
 set(SCREAM_MIMIC_GPU ${DEFAULT_MIMIC_GPU} CACHE BOOL "Mimic GPU to correctness-test inter-column parallelism on non-GPU platform")
 set(SCREAM_PACK_CHECK_BOUNDS FALSE CACHE BOOL "If defined, scream::pack objects check indices against bounds")
 set(SCREAM_TEST_DATA_DIR ${CMAKE_CURRENT_BINARY_DIR}/data CACHE PATH "Location of data files generated by tests")
-set(SCREAM_MPI_ERRORS_ARE_FATAL TRUE CACHE BOOL "Whether MPI errors should abort (default TRUE). If false, errors should be handled.")
```

Comment:
I think this got deleted by accident.

Reply:
> Remove unused SCREAM_MPI_ERRORS_ARE_FATAL

Reply:
Why did you choose to remove it?

Reply:
It wasn't being used anywhere.

Reply:
To clarify, the string SCREAM_MPI_ERRORS_ARE_FATAL only occurs here, so it doesn't do anything.

Reply:
No, it is used to set the default for EKAT_MPI_ERRORS_ARE_FATAL, which is used in the `ekat_comm.*pp` files. It's true we don't make use of this feature so far, but one may set `SCREAM_MPI_ERRORS_ARE_FATAL=OFF` in its config, and things would change.

Reply:
Yeah, it's confusing. There's nothing wrong with scream builds setting ekat stuff directly. Should I restore this parameter or keep it removed?